src/Grids/grid_utils.jl

```diff
@@ -27,6 +27,8 @@ Base.size(loc, grid, d) = size(loc, grid)[d]
 
 total_size(a) = size(a) # fallback
 
+halo_size(grid) = (grid.Hx, grid.Hy, grid.Hz)
```

Comment:
This already exists in 

https://github.com/CliMA/Oceananigans.jl/blob/3636a2ed2c54723cb0523ac02b4817187c1fdebd/src/Grids/Grids.jl#L138

I suggests we remove it from `Grids.jl` and only keep it here!